<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_del">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Delete satellites</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_delSats">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="examp_del">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string> Example: G01;C01;R01;E01</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Obs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>721</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>SYS/#/OBS TYPES (Set the PPP dual-frequency observation type)</string>
   </property>
   <widget class="QLabel" name="gps_label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>29</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GPS_L1">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>24</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GPS_L2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>24</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GLO_L1">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>24</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GLO_L2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>24</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="glo_label">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>27</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>GLONASS:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="BDS_L2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>23</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="BDS_L1">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>23</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bds_label">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>26</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BDS:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GAL_L2">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>22</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GAL_L1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>22</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gal_label">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>25</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Galileo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="examp_obs">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>55</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Example: GPS: L1C L2W</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_del_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>721</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Set Parameters(m^2/30s)</string>
   </property>
   <widget class="QPlainTextEdit" name="Qw_pos">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>40</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QLabel" name="Qw_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>45</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer noise (Qw):</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_zwd">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3e-8</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pk_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial covariance (Pk):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>14</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>dynamic_pos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>14</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>zwd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>14</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>clk</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_clk">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>568</x>
      <y>14</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>amb</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_amb">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e-16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>14</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ion</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_ion">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_amb">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_pos">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>80</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_ion">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_clk">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_zwd">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>123</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrier/Pseudorange precision (m):</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="L_precision">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>120</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.003</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="P_precision">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
